examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/README.md

```diff
@@ -0,0 +1,113 @@
+# Mina zkApp: Zkapp Offchain Storage
+
+An experimental library for storing zkApp data offchain. This project is currently under development and should not be used in production.
+
+## Usage
+
+### Dependencies:
+
+Ensure all required modules are installed. To do so, run the following command:
+
+```bash
+npm install
+```
+
+### Running the Example
+
+#### Blockchain Mode
+
+Toggle the `useLocalBlockchain`` variable to switch between local testing and real-world usage.
+
+#### Account Configuration:
+
+For local testing, predefined accounts will be used.
+For `Berkeley` mode, you must provide a `deployerPrivateKey` and `zkAppPrivateKey` as command line arguments.
+
+To generate these keys, you can use the following command:
+
+```bash
+npm run generate-keys
```

Comment:
```suggestion
npm run generate-key
```

Reply:
The script in the package.json is `npm run generate-key`

Reply:
@ymekuria  I see what you did there: GitHub suggesting... love it

Reply:
Thanks for showing me the way @barriebyron!